tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -17,16 +17,16 @@ set(test_src
     Fixtures/SQLiteFixture.hh
     PhaseChangeModeling/HysteresisModel.unit.cc
     AdvancedAFN.unit.cc
-    AirflowNetworkSimulationControl.unit.cc
```

Comment:
I added one new file here, but the rest of the changes are just due to sorting the lines.